tasks/build/index.js

```diff
@@ -1,35 +1,52 @@
 "use strict"
 
 const { cli } = require(`@nodeguy/cli`)
-const options = require(`./options.json`)
+const specification = require(`./options.json`)
```

Comment:
what is your thinking here @NodeGuy ?

Reply:
I just clarified it.

Reply:
I mean I pushed a new commit that renames `specification` to `optionsSpecification`.  Does that help?